packages/@okta/i18n/src/properties/login.properties

```
@@ -804,6 +804,7 @@ oie.enroll.okta_verify.select.channel.qrcode.label = Scan a QR code
 oie.enroll.okta_verify.select.channel.email.label = Email me a setup link
 oie.enroll.okta_verify.select.channel.sms.label = Text me a setup link
 oie.enroll.okta_verify.select.channel.title = More options
+oie.enroll.okta_verify.email.label = Email
```

Add this since there is no Email Label key, don't think we want to reuse the `Email`  for email authenticator name, in case of the name changes later.

---

super-nit: Can we keep the existing convention of nesting the form name in here as well?

```suggestion
oie.enroll.okta_verify.channel.email.label = Email
```